### DISCUSSION TITLE
Shift_factor is not used in sd3 training.

### DISCUSSION THREAD

**Discussion by laksjdjf** [huggingface/diffusers#8674] [Q&A]
In img2img pipeline, vae'output substracted by shift_factor.
https://github.com/huggingface/diffusers/blob/f1f542bdd4412a97ac109dab22959a8a575f3a7e/src/diffusers/pipelines/stable_diffusion_3/pipeline_stable_diffusion_3_img2img.py#L629

But not substracted in training.
https://github.com/huggingface/diffusers/blob/f1f542bdd4412a97ac109dab22959a8a575f3a7e/examples/dreambooth/train_dreambooth_sd3.py#L1547

Is thig bug?

**Comment by laksjdjf** [comment]
it seem to be same as https://github.com/huggingface/diffusers/issues/8708 .

**Comment by sayakpaul** [comment]
Fixed https://github.com/huggingface/diffusers/pull/8917
